src/com/xilinx/rapidwright/design/DesignTools.java

```diff
@@ -3613,4 +3617,243 @@ public static boolean isNetDrivenByHierPort(Net net) {
 
         return true;
     }
+
+    /**
+     * Locks the logical netlist of the design using the DONT_TOUCH property.
```

Comment:
I think it would be documenting whether this (and the following methods) are identical/not identical to Vivado `lock_design`?